File: RA1/JAVA/EjemploHerencia/Cow.java
package EjemploHerencia;

public class Cow extends Animal{
    private float hornSize;

    public Cow(String name, float hornSize){
        super(name);//llamar al constructor de la clase superior, es decir, de clase Animal
        this.hornSize = hornSize;
    }
    
    @Override
    public void makeSound(){
        System.out.println(this.name + " brama: Muuu!!!");
    }
}

File: RA1/PYTHON/ejemplo_herencia/cow.py
#from <archivo> import <elemento<class>>
from typing import overload
from animal import Animal

class Cow(Animal):
    def __init__(self, name, horn_size):
        #opcion 1: 
        super().__init__(name)
        #opcion 2: Animal.__init__(name)
        self.horn_size = horn_size

    def make_sound(self):
        print(self.name + " brama: MUUU...")

    